src/distributions/mod.rs

```diff
@@ -176,7 +178,7 @@ fn ziggurat<R: Rng+?Sized, P, Z>(
                 } else {
                     bits.closed_open01_fixed()
                 };
-        let i = (bits & 0xff) as usize;
+        let i = ((bits >> 3) & 0xff) as usize;
```

Comment:
Were there two unused bits? I thought you used them all?

Reply:
There are 11 spare bits, and we only need 8